API/projectClass/contrastsClass.m

```diff
@@ -234,16 +238,49 @@
                 addParameter(p,'domainRatio',   defaultDomainRatio, @(x) any(validatestring(x,expectedDomainRatio)));
             end
 
+            addParameter(p,'model',            defaultModel, @(x) validateContrastModel(x,modelType,expectedModel));
+
+            % Set up the validators
+            function validateExactString(input, allowedNames)
```

Comment:
Why are these functions nested? The could be private with no loss of functionality

Reply:
I'll look again, but I tried having them as private and couldn't get it to work - I got an error saying it couldn't find the function.